std/parallelism.d

```diff
@@ -2633,7 +2633,7 @@ public:
 
             // Hack to take the address of a nested function w/o
             // making a closure.
-            static auto scopedAddress(D)(scope D del)
+            static auto scopedAddress(D)(scope D del) @system
```

Comment:
I pulled this as this function returns its scoped variable, so it can't be `@safe` plus it doesn't modify the behavior of Phobos and helps Walter to move on.
For the discussion on the behavior of attribute inference, please see https://github.com/dlang/dmd/pull/5972#issuecomment-242227591